### DISCUSSION TITLE
what's the syncOptions for application

### DISCUSSION THREAD

**Discussion by 13567436138** [argoproj/argo-cd#6680] [Q&A]
what's the syncOptions for application.
can you give me some example of syncOptions

**Comment by cnukwas** [comment]
Is this what you're looking for about [sync options](https://argoproj.github.io/argo-cd/user-guide/sync-options/)?

**Reply by jgwest** [reply]
Also see [this YAML Application example](https://github.com/argoproj/argo-cd/blob/e17565ae8124706feb7c17b23195885d8173a26b/docs/operator-manual/application.yaml#L100) which includes an example of the fields used within an Application resource.
